### ISSUE TITLE
Open source alternative to snap

### ISSUE THREAD

**Issue by ShalokShalom** [anbox/anbox#1361]
Hi, my distribution does not support snap and its a closed source solution, server wise. 
Could you consider an open-source friendly alternative, additionally? 
Thanks a lot :D

**Comment by Fuseteam** [comment]
have you tried installing snapd? as for alternatives i believe that question is answered on [anbox.io](https://anbox.io/#collapse8)

**Comment by mainmachine** [comment]
You can easily run snaps built locally as long as you have snapd installed, which is of course FOSS. The README.md explains exactly how to build the snap yourself, install and run it: https://github.com/anbox/anbox/blob/master/README.md

**Comment by Fuseteam** [comment]
@mainmachine that........makes me wonder what part of the server is closed sourced if snaps can be build locally

**Comment by mainmachine** [comment]
@Fuseteam - Some of the folks at Canonical that work on snaps, snapd and snapcraft.io have addressed your concern on one of the popular Linux podcasts... it was either the [Ubuntu podcast](https://ubuntupodcast.org/) or [Late Night Linux](https://latenightlinux.com/) at some point last year? I believe it was Alan Pope that addressed it. It's not "closed source", they have chosen not to release the source yet - that's not licensing, it's distribution of code (or an intentional lack of, in this case)

**Comment by Fuseteam** [comment]
@mainmachine not really an concern of mine but of the OP, let me rephrase my question:
if snaps can be build locally what part of the source code is not released yet?

**Comment by ShalokShalom** [comment]
That one that offers alternatives to snap store and I honestly question the decision in the first place, because this is not very open source friendly to begin with from both Canonical and Anbox.

Why supporting Canonicals approach to close sourcing this - and this was intentional to limit the avaiable sources of snaps to really only their own service, since there are half a dozen proper true open source alternatives?

Also, my distribution dont support snap at all.

**Comment by Fuseteam** [comment]
@ShalokShalom i have trouble understanding where you see a problem. snaps current intention is be a "google play for desktop linux" to attract more devs to linux desktop

fwiw snap isn't close sourcing anything the code is still fully and freely available. heck the readme even tells you how to build the snap from source. the reason snap was used for this project is because it is simply easier on the developer to both distribute and update anbox this way.

what distro are you on? it is likely you just need to install snapd with your package manager to "support snaps"

**Comment by ShalokShalom** [comment]
The snap store itself is non free.
There are a gazillion alternatives like Flatpak and Habitat.
KaOS. Snap is not packaged here. As said, not supported.

Thanks for nothing, you just propose a non free solution.

**Comment by mainmachine** [comment]
@ShalokShalom - I suggest you fork Anbox and refactor it for a distribution method that you find preferable to snap.

**Comment by Fuseteam** [comment]
@ShalokShalom for the record i'm not propposing anything i'm simply explaining how snaps work from my understanding, why i say it's not "closed sourced" , why the developer chose snap for anbox and why they chose not to release the code for the snap store yet :p

i already proposed pr'ing whatever is needed for flatpak and that the dev is open for packaging flatpak if anyone is up for it
